app/views/shared/_footer.html.erb

```diff
@@ -56,6 +56,14 @@
             <%=t("views.public_pages.privacy_policy.title") %>
           <% end %>
         </li>
+
+        <% unless Rails.env.production? %>
+          <li>
+            <%= link_to new_user_session_path do %>
+              <%= t("views.shared.footer.volunteer_sign_in") %>
+            <% end %>
+          </li>
+        <% end %>
```

Comment:
This allows demo users to use the sign in form

Reply:
✨